agent/transformers/graphite.go

```diff
@@ -0,0 +1,53 @@
+package transformers
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	"github.com/sensu/sensu-go/types"
+)
+
+// Graphite contains values of graphite plain text metric format
+type Graphite struct {
+	Path      string
+	Value     float64
+	Timestamp int64
+}
+
+// Transform transforms a metric in graphite plain text format to Sensu Metric
+// Format
+func (g Graphite) Transform() []types.MetricPoint {
+	mp := types.MetricPoint{
+		Name:      g.Path,
+		Value:     g.Value,
+		Timestamp: g.Timestamp,
+		Tags:      []*types.MetricTag{},
+	}
+	return []types.MetricPoint{mp}
+}
+
+// ParseGraphite parses a graphite plain text string into a Graphite struct
+func ParseGraphite(metric string) (Transformer, error) {
```

Comment:
It's better to return a Graphite type here. It will be a Transformer since it implements the Transformer method set.

Reply:
You got it!